plugins/inputs/fritzbox/sample.conf

```
@@ -0,0 +1,30 @@
+# Gather fritzbox status
+[[inputs.fritzbox]]
+  ## The URLs of the devices to query. For each device the corresponding URL
+  ## including the login credentials must be set. E.g.
+  ## urls = [
+  ##   "http://boxuser:boxpassword@fritz.box:49000/",
+  ##   "http://:repeaterpassword@fritz.repeater:49000/",
+  ## ]
+  urls = [
+  ]
+
+  ## The information to collect (see README for further details).
+  collect = [
+    "device",
+    "wan",
+    "ppp",
+    "dsl",
+    "wlan",
+  ]
```

It is the default, so it can be commented out?
```suggestion
  # collect = [
  #   "device",
  #   "wan",
  #   "ppp",
  #   "dsl",
  #   "wlan",
  # ]
```

---

Done via 8e18f42caf88c7a9d483c8681b2c6b310317a022